packages/mux-player/src/html.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  TemplateInstance,
+  NodeTemplatePart,
+  createProcessor,
+  AttributeTemplatePart,
+} from "@github/template-parts";
+import type { TemplatePart, TemplateTypeInit } from "@github/template-parts";
+
+function processSubTemplate(part: TemplatePart, value: unknown): boolean {
+  if (value instanceof TemplateResult && part instanceof NodeTemplatePart) {
+    value.renderInto(part);
+    return true;
+  }
+  return false;
+}
+
+function processDocumentFragment(part: TemplatePart, value: unknown): boolean {
+  if (value instanceof DocumentFragment && part instanceof NodeTemplatePart) {
+    if (value.childNodes.length) part.replace(...value.childNodes);
+    return true;
+  }
+  return false;
+}
+
+export function processPropertyIdentity(
+  part: TemplatePart,
+  value: unknown
```

Comment:
non-blocking: We should be able to use `<T = any>processPropertyIdentity(part: TemplatePart, value: T)` to make this a little better with types, especially if we ever get into a case where we want to handle different types differently.